applications/ContactStructuralMechanicsApplication/custom_strategies/custom_convergencecriterias/displacement_lagrangemultiplier_contact_criteria.h

```diff
@@ -150,7 +150,7 @@ class DisplacementLagrangeMultiplierContactCriteria
         mLMRatioTolerance =  ThisParameters["contact_displacement_relative_tolerance"].GetDouble();
         mLMAbsTolerance =  ThisParameters["contact_displacement_absolute_tolerance"].GetDouble();
 
-        // Additional flags -> NOTE: Replace for a ral flag?¿
+        // Additional flags -> NOTE: Replace for a real flag?¿
```

Comment:
strange character needed?
```suggestion
        // Additional flags -> NOTE: Replace for a real flag?
```

Reply:
I would say non ASCII character ¿ is not strange In Spanish, you needed to start a question (which is very practical when reading a long question in my opinion)